builtin/providers/aws/resource_aws_dhcp_option_set.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mitchellh/goamz/ec2"
+)
+
+func resourceAwsDhcpOptionSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDhcpOptionSetCreate,
+		Delete: resourceAwsDhcpOptionSetDelete,
```

Comment:
I think we are missing a Refresh function. That should map to DescribeDhcpOptions. Otherwise we don't know if the DhcpOptions have been deleted or modified for some reason